### ISSUE TITLE
NTR: Vasospasm

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#4708]
**Preferred term label:**
Vasospasm

**Synonyms**
Blood vessel spasm, Vasospasm, Angiospasm, Vascular spasm

**Definition (free text, please give PubMed ID)**
Narrowing of arteries due to constriction of the blood vessels ([PMID:20557770](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3328054/)).

**Parent term (use hpo.jax.org/app)**
HP_0030163 Abnormal vascular physiology

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Raynaud's disease [OMIM:179600](https://www.omim.org/entry/179600?search=Raynaud%27s&highlight=raynaud)

**Your nano-attribution (ORCID)**
ORCID: [0000-0002-8169-9049](https://orcid.org/0000-0002-8169-9049)

**Comment by pnrobinson** [comment]
added
